lte/gateway/python/magma/pipelined/app/uplink_bridge.py

```diff
@@ -527,6 +531,8 @@ def _start_dhclient_if(self, if_name: str, af_inet: int):
         setup_dhclient = ["dhclient", if_name]
         try:
             subprocess.check_call(setup_dhclient)
+            # delay to get DHCP address
+            hub.sleep(2)
```

Comment:
what does this delay do? seems like the call above is already blocking/waiting?

Reply:
The delay is for DHCP transaction to complete.